massa-wallet/src/lib.rs

```diff
@@ -77,8 +77,9 @@ impl Wallet {
         }
     }
 
-    pub fn remove_address(&mut self, address: Address) -> Option<(PublicKey, PrivateKey)> {
-        self.keys.remove(&address)
+    pub fn remove_address(&mut self, address: Address) -> Result<(), WalletError> {
+        self.keys.remove(&address).ok_or(WalletError::MissingKeyError(address))?;
+        self.save()
     }
 
```

Comment:
also need to run "cargo fmt" before pushing for sanity

Reply:
otheerwise, all good !

Reply:
You can set up you code editor to run cargo fmt on save

Reply:
Yes I have made it but only on the massa_client subfolder. Done.